pkg/identity/identity.go

```diff
@@ -76,3 +80,11 @@ func (id *signingIdentity) Credentials() []byte {
 func (id *signingIdentity) Sign(message []byte) ([]byte, error) {
 	return id.sign(message)
 }
+
+func (id *signingIdentity) Serialize() ([]byte, error) {
```

Comment:
This should not need to be part of the public API

Reply:
> This should not need to be part of the public API

the code here and Serialize function  has been removed.